tests/index.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+require '../includes/functions.php';
+
+use PHPUnit\Framework\TestCase;
+
+class Index extends TestCase
+
+{
+    public function testSuccessDisplayCard()
+        // my input is an array and my expected output is a string
+    {
+        $input = ['club' => 'dog', 'player_first_name' => 'cat', 'player_last_name' => 'bird', 'image' => 'deer', 'position' => 'rabbit', 'DOB' => 'turtle', 'height' => 'wolf', 'weight' => 'gorilla'];
+        $expectedOutput = '<div class="card dog"><div class="card_top"><h2 class="player_name">cat bird</h2><h3 class="club">dog </h3></div><img src="deer"><div class="stats"><p class="position">Position: rabbit </p><p class="DOB">DOB: turtle </p><p class="height">Height: wolf m </p><p class="weight">Weight: gorilla kg </p></div></div>';
+
+        $result = display_card($input);
+
+        $this->assertEquals($expectedOutput, $result);
+    }
+
+    public function testFailureDisplayCard()
```

Comment:
its good practise to specify which field you are testing here. e.g:
`testFailureDisplayCard_missingClub()`